### ISSUE TITLE
`Export` Variable Action on Airflow UI not available on Astronomer

### ISSUE THREAD

**Issue by paolaperaza** [astronomer/astronomer#431]
**Describe the bug**

An Enterprise customer on Astronomer v0.9.5 and Airflow 1.10.3 is seeing an inconsistency between his local and remote Airflow deployments re: the `Export` Variable action in the Airflow UI.

**To Reproduce**

1. As a Workspace Admin, create a new deployment (either Local or Celery) on app.datarouter.ai
2. Navigate to `Admin` > `Variables` in the Airflow UI
3. Click on `Actions` menu, and see `Delete`, but not `Export`
4. Spin up a local Airflow deployment with the same Astro + Airflow versions, and see the `Export` option

Tested this on:`Astronomer v0.10.0-rc.8` and `Airflow v1.10.4`, but a customer reported it on `Airflow 1.10.3` and `Astronomer v0.9.5`.

**Expected behavior**

A local Airflow environment and a deployment on Astronomer should exhibit parallel functionality given the same Airflow and Astronomer versions + a user with `Admin` status on Astronomer.

**Screenshots**
Airflow on Astronomer:

<img width="389" alt="Screen Shot 2019-08-19 at 11 53 17 AM" src="https://user-images.githubusercontent.com/31111752/63303770-d0853680-c2ae-11e9-9394-b49aa5b95a41.png">

Localhost:

![image (9)](https://user-images.githubusercontent.com/31111752/63303782-dbd86200-c2ae-11e9-868f-6fd53619e81e.png)

**Additional context**

@ashb Think this might be something related to how we implemented RBAC?

**Comment by ashb** [comment]
Yeah, locally he is an Admin user, but in the cluster we make people Operator (`Op`) users only. So the difference is currently to be expected.

Do we want to enable Variable export?
